### DISCUSSION TITLE
dfint-steam-hook-0.1.8.zip detect as a virus

### DISCUSSION THREAD

**Discussion by Alarhon** [dfint/.github#11] [Q&A] [answered]
dfint-steam-hook-0.1.8.zip is detecting as a virus is that a false positive? Sorry for my bad english isn't my mother tongue.

**Comment by insolor** [comment, marked as answer]
The hook intrudes to the game's memory and code itself to make it show text in other language. Of course, some antiviruses see this as a suspicious behavior, but this is just the way it works. So, this is an expected reaction from antiviruses.

Don't apologize that English is not your mother tongue, this is actually the reason we all are here.
